### ISSUE TITLE
Record audio as a separate file

### ISSUE THREAD

**Issue by jestinjoy** [MaartenBaert/ssr#760]
Is it possible to save audio as a separate file ?

**Comment by MaartenBaert** [comment]
No, but you can extract audio easily from the video file. E.g. in Audacity, you can simply import the video file to get the audio, and then export it again to an audio file.

**Comment by HybridDog** [comment]
With ffmpeg it is possible to separate the video and audio data, and more.
For example, if the audio is in aac format, it can probably be extracted losslessly with this (untested) command:
```ffmpeg -i video.mp4 -c copy audio.aac```
